content/docs/next-release-v2/apis.md

```diff
@@ -121,7 +121,10 @@ Jaeger UI communicates with **jaeger-query** Service via JSON API. For example,
 
 **Status**: Stable
 
-When using the `grpc` storage type (a.k.a. [remote storage](../storage/#remote-storage)), Jaeger components can use custom storage backends as long as those backends implement the gRPC [Remote Storage API][storage.proto].
+When using the `grpc` storage type (a.k.a. [remote storage](../storage/#remote-storage)),
```

Comment:
the text should make distinction between v1 and v2 remote APIs. I think v1 is stable but v2 is kinda still experimental.

Reply:
@yurishkuro What kind of a distinction do we want to make here? This is in the v2 docs

Reply:
You're right. Did we indicate a breaking change that v2 no longer supports v1 remote API?